controllers/vm_controller.go

```diff
@@ -65,13 +66,20 @@ func (r *VmReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Re
 
 			return ctrl.Result{}, nil
 		}
-		r.log.Error(err, "Error getting VM", "vm", req.NamespacedName)
-		return ctrl.Result{}, err
+
+		r.log.Error(err, "could not find VM", "vm", req.NamespacedName)
+		return ctrl.Result{
+			Requeue:      true,
+			RequeueAfter: 5 * time.Second,
```

Comment:
@machadovilaca  Why do we wait here 5sec and for the Volumes we wait 30sec?

Reply:
waiting time for the PVC to be create, but not needed anymore